changelog/64322.removed.md

```diff
@@ -0,0 +1 @@
+Removing Azure-Cloud modules from the code base.
```

Comment:
Can we elaborate how users can now access azure cloud with the Salt extension. 

I think this is definitly something we should really highlight in the RElease notes file since this is an impactful change. You will just need to edit `doc/topics/releases/templates/3007.0.md.template`